server/internal/core/application/types.go

```diff
@@ -70,6 +71,13 @@ type ServiceInfo struct {
 	Dust                       uint64
 	BoardingDescriptorTemplate string
 	ForfeitAddress             string
+	MarketHour                 *NextMarketHour
```

Comment:
this struct isn't really needed to me, we can use a `domain.MarketHour`?

Reply:
MarketHour is domain what is stored in DB, NextMarketHour(contains NextStartTime) is calculated based on MarketHour, imo its logic is more clear with this.